src/components/shared/Layer.js

```diff
@@ -0,0 +1,13 @@
+import { Layer as GrommetLayer } from 'grommet'
+import { useResponsive } from 'hooks/useResponsive'
+
+export const Layer = ({ position = '', show = false, children }) => {
+  const { viewport } = useResponsive()
+  if (viewport !== 'small') return children
+  return (
+    // eslint-disable-next-line react/jsx-no-useless-fragment
+    <>{show && <GrommetLayer position={position}>{children}</GrommetLayer>}</>
```

Comment:
This rule exception doesn't need to be applied here. Your component can have two returns, if `show` is false return before the markup

Reply:
This component is shared with other components (e.g., [search result's filter](https://github.com/AlexsLemonade/refinebio-web/blob/efe966bfd19d5a9fa2e5925f6cfcdc3b2fdaff23/src/pages/search/index.js)
), and for some cases, the `show` state evaluates whether or not its content is toggled (by default, it's hidden). So fir this reason, simply returning the `children` when `show` is false won't work.

Reply:
added a suggestion below

Reply:
Resolved in https://github.com/AlexsLemonade/refinebio-web/pull/31#discussion_r1120231495